mandatory/3-playing-computer.js

```diff
@@ -28,7 +28,7 @@ const f2 = function (a, b) {
 
 console.log(x);
 console.log(a);
-console.log(b);
+//console.log(b);
 
 for (let i = 0; i < 5; ++i) {
```

Comment:
Please provide answers, to the questions asked in this exercise. It is a great way for mentors to support your effort.